src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -70,8 +69,7 @@ class OnfidoStep extends React.Component {
                     this.canShowOnfido() ? (
                         <Onfido
                             sdkToken={this.props.walletOnfidoData.sdkToken}
-                            onError={(error) => {
-                                Log.hmmm('Onfido error in OnfidoStep', {error});
+                            onError={() => {
```

Comment:
Unsure if this is a priority given the safe assumption that _most_ users are probably using web to go through Onfido and not native. 

But if this (and the `onError` in `RequestorStep`) were logging errors on native before I think it's not anymore?

https://github.com/Expensify/App/blob/4b7f78a703daa71a083384d757065fd6e79f8517/src/components/Onfido/index.native.js#L35-L45

Reply:
I will approve and let you decide if you think this is valuable. I have a feeling it's more valuable to cover the mainline case immediately and then worry about native if/when we see more users engaging with the flow there.